test/integration/webhookgithub_test.go

```diff
@@ -299,6 +300,11 @@ func postFile(client restclient.HTTPClient, event, filename, url string, expStat
 	if resp.StatusCode != expStatusCode {
 		t.Errorf("Wrong response code, expecting %d, got %d: %s!", expStatusCode, resp.StatusCode, string(body))
 	}
+	var newBuild buildapi.Build
+	if err = runtime.DecodeInto(kapi.Codecs.UniversalDecoder(), body, &newBuild); err != nil {
```

Comment:
you have the bytes, so you should unmarshal with json.unmarshal (never do this in "real" code), into a v1 build object and ensure you have content (more than just metadata).